RobertMrowiec/CarBIDEmber
# Auctions per user

# Tasks:
- create myauction list,
- allow user to edit his auction

# Testing scenario:
1. login
2. add auction
3. click my auctions from navbar
4. check how it works / edit your created auction

# Preview
![image](https://user-images.githubusercontent.com/25043084/56896027-f9c55100-6a8a-11e9-9ba6-8419cf71e7bb.png)
